src/lib-components/GridGallery.vue

```diff
@@ -0,0 +1,230 @@
+    type: Boolean,
+    default: false
+  }
+})
+
+const parsedItems = computed(() => {
+  return items.map((obj) => {
+    let objClass = 'card '
+    if (obj.featured && obj.snippet)
+      objClass = `${objClass}card-featured`
+    else if (obj.featured && !obj.snippet)
+      objClass = `${objClass}card-4up`
+    else
+      objClass = `${objClass}card-small`
+
+    return {
+      ...obj,
+      classes: objClass,
+    }
+  })
+})
+
+const parsedAspectRatio = computed(() => {
+  return featured ? 100 : 60
+})
+</script>
+
+<template>
+  <div class="grid-gallery">
+    <ul class="cards">
+      <li
+        v-for="card in parsedItems"
+        :key="card.to"
+        :class="card.classes"
+      >
+        <ResponsiveImage
+          :media="card.image"
+          :aspect-ratio="parsedAspectRatio"
+          object-fit="cover"
+        />
+        <h5
+          v-if="
+            card.headlineText
+              && ((card.featured && !card.snippet) || !card.featured)
+          "
+          class="title"
+        >
+          <SmartLink
+            :to="card.to"
+            link-target="_blank"
+            class="link"
+          >
+            {{ card.headlineText }}
+          </SmartLink>
+        </h5>
+        <div
+          v-if="!card.featured && card.snippet"
+          class="snippet"
+          v-html="card.snippet"
+        />
+        <div v-if="card.featured && card.snippet" class="section-text">
+          <div class="text">
+            <h5 v-if="card.headlineText" class="title">
+              <SmartLink
+                :to="card.to"
+                link-target="_blank"
+                class="link"
+              >
+                {{ card.headlineText }}
+              </SmartLink>
+            </h5>
+            <div class="snippet" v-html="card.snippet" />
+          </div>
+        </div>
+      </li>
+    </ul>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+.grid-gallery {
+  max-width: $container-l-main + px;
+  padding: 0;
+  margin: 0;
+  background-color: var(--color-white);
+  position: relative;
+
+  .section-summary {
+      @include step-1;
+      color: var(--color-black);
+      position: sticky;
+      top: 48px;
+      background-color: var(--color-white);
+      z-index: 20;
+      padding-bottom: 16px;
+      padding-top: 10px;
+      // margin-top: var(--space-xl);
+  }
+  .cards {
+      display: grid;
+      grid-gap: 16px;
+      grid-template-columns: repeat(6, 1fr);
+      grid-auto-flow: dense;
+      margin-top: var(--space-m);
+  }
+  .card {
+      margin-bottom: var(--space-l);
+      list-style: none;
+      padding: 0;
+      position: relative;
+      &:last-child {
+          margin-bottom: 0;
+      }
+  }
+  .card-small {
+      grid-column: span 2;
+  }
+
+  .card-featured {
+      grid-column: span 6;
+      position: relative;
+
+      display: grid;
+      grid-template-columns: 1fr;
+      gap: 0;
+      grid-column: span 6;
+
+      .two-librarians {
+          :deep {
```

Comment:
should this be :deep(.media), but also two-librarians is not in this component, so may be it needs to be :deep(.two-librirans) {
.media {
?

Reply:
@pghorpade I don't know why this selector is here either. As a precaution, I don't delete selectors in case they may have an (intended) effect somewhere else. I just checked the main repo and this is the only reference of it. So maybe it should be deleted?

Reply:
@pghorpade I did a wider (org) search and two-librarians is a class coming from this .json file in the nuxt-site: https://github.com/UCLALibrary/library-website-nuxt/blob/d1020b83717c3b409ac68a2edc513579ec0e38ee/data/impact-report_index.json#L89